indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -100,16 +100,19 @@
 
         <div ng-app="nd" ng-controller="AppCtrl">
             <div nd-reg-form
-                conf-id="{{ event.id }}"
-                conf-sections="{{ sections | tojson | forceescape }}"
-                conf-currency="{{ regform.currency }}"
-                user-info="{{ user_data | tojson | forceescape }}"
-                post-url="{{ request.url | tojson | forceescape }}"
-                check-email-url="{{ url_for('.check_email', regform, management=management) | tojson | forceescape }}"
-                management="{{ management | tojson | forceescape }}"
-                is-moderated="{{ moderated | tojson | forceescape }}"
-                notify-default="{{ session.get('registration_notify_user_default', true) | tojson | forceescape }}">
+                 conf-id="{{ event.id }}"
+                 conf-sections="{{ sections | tojson | forceescape }}"
+                 conf-currency="{{ regform.currency }}"
+                 user-info="{{ user_data | tojson | forceescape }}"
+                 post-url="{{ request.url | tojson | forceescape }}"
+                 check-email-url="{{ url_for('.check_email', regform, management=management) | tojson | forceescape }}"
+                 management="{{ management | tojson | forceescape }}"
+                 is-moderated="{{ moderated | tojson | forceescape }}"
+                 notify-default="{{ session.get('registration_notify_user_default', true) | tojson | forceescape }}">
             </div>
         </div>
+        {% if not management %}
+            {{ template_hook('cern-access-personal-data', event=event, regform=regform, form=form) }}
```

Comment:
this is way too specific for a template hook... find a generic name, pass `management` as a kwarg, and don't put cern-specific code in the core...